order-system-ionic/src/pages/Cart/Cart.tsx

```diff
@@ -59,35 +60,7 @@ const Cart: React.FC = () => {
             </IonButton>
           </div>
 
-          <Divider />
-
-          {cartData.items.map((foodItem, index) => (
-            <TextCard
-              key={foodItem.item.id}
-              lines={index === cartData.items.length - 1 ? "" : "none"} // Line only for the last item
-              label={`${foodItem.quantity}x ${foodItem.item.name}`}
-              note={`${(foodItem.item.price * foodItem.quantity).toFixed(2)}`}
-              noteColor="black"
-            />
-          ))}
```

Comment:
I think its okay to combine them in the payment as the cards as the top are like receipts that break down the costs. In the cart the cards at the top are only the items with no price breakdown so it would be nice to have the price breakdown at the bottom

Reply:
ok added it back